acceptance/testdata/mock_buildpacks/v1/simple-layers-buildpack/buildpack.toml

```diff
@@ -0,0 +1,9 @@
+schema = "1"
```

Comment:
do we need to keep some documentation of the previous schema versions? Is `schema` a required element? If not, what is the default when it's not present? We could probably sort this all out in an RFC.

Reply:
Will be removing this for this PR